src/gwrefpy/__init__.py

```diff
@@ -1,7 +1,23 @@
+from .constants import print_constants
 from .methods.timeseries import analyze_offsets
 from .model import Model
+from .utils import (
+    datetime_to_float,
+    enable_file_logging,
+    float_to_datetime,
+    set_log_level,
+)
 from .well import Well
 
 __name__ = "gwrefpy"
 __version__ = "0.1.0"
```

Comment:
```suggestion
__version__ = "0.2.0"
```
Update to version 0.2.0